app/config.py

```diff
@@ -52,8 +52,12 @@ class QueueNames(object):
     # pretty quickly.
     SEND_NORMAL_QUEUE = "send-{}-tasks"  # notification type to be filled in the queue name
 
-    # Queue for sending all SMS, except long dedicated numbers.
-    # TODO: Deprecate to favor priority queues instead, i.e. bulk, normal, priority.
+    # Queues for sending all SMS, except long dedicated numbers.
+    SEND_SMS_HIGH = "send-sms-high"
```

Comment:
I am not asking to change this - but asking a question:

why did we choose sms_high, sms_medium, sms_low instead of sms_priority, sms_bulk, sms_normal so it maps perfectly to our current system?

Reply:
We want to refactor internally for some time the names because these are confusing. The current names at the moment are more aligned with the use case rather than their speed intent (bulk vs low priority, it's a use case vs speed name, or saying _bulk priority_ in itself when bulk has no priority, so saying low priority would be more sound, or saying _it is priority_ to mean high priority but we already have normal and bulk priority, etc..). Steve most likely named these in anticipation for the future refactoring.

What do you think about these? We can have a chat and sync on this.

Reply:
yeah, I was going to just copy what we did before (as I did for much of how the new code is written) but really really hate how we do it (basically what jimmy said), so thought I'd just go with high / med / low and spark the conversation 😁

Reply:
spoke offline. this looks good.